apps/system/style/chrome/chrome.css

```diff
@@ -386,6 +392,8 @@
 .controls .back-button[disabled],
 .controls .forward-button[disabled] {
   display: none;
+  opacity: 0;
+  animation: fadeOut .2s ease-in;
```

Comment:
It would visually work, but we need to display:none at the end, as the buttons are relatively positioned.

Regarding fade out, you are right, is not working properly now. To be honest, I think I'm going to remove both animations, as the fadeIn is not adding a lot of value...